repos/fdbt-site/src/pages/ticketRepresentation.tsx

```diff
@@ -82,23 +84,29 @@ const TicketRepresentation = ({
                         <fieldset className="govuk-fieldset" aria-describedby="ticket-representation-page-heading">
                             <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
                                 <h1 className="govuk-fieldset__heading" id="ticket-representation-page-heading">
-                                    {`Select a type of ${fareTypeDesc} ticket`}
+                                    {`Select a type of ${sentenceCaseString(fareTypeDesc)} ticket`}
                                 </h1>
                             </legend>
                             <FormElementWrapper errors={errors} errorId="geo-zone" errorClass="govuk-radios--errors">
                                 <RadioButtons
                                     inputName="ticketType"
                                     options={[
-                                        {
-                                            value: 'geoZone',
-                                            label: 'A ticket within a geographical zone',
-                                            hint: fareTypeHint.geoZone,
-                                        },
+                                        ...(showGeoZone && isDevOrTest
```

Comment:
How come we're using isDevOrTest here?

Reply:
i was thinking, it will be feature flagged

Reply:
No need to be feature flagged unless the ticket says so